test/alternator/conftest.py

```diff
@@ -199,7 +204,10 @@ def dynamodb_test_connection(dynamodb, request, optional_rest_api):
         response = requests.get(url, verify=False)
         assert response.ok
     except:
-        pytest.exit(f"Scylla appears to have crashed in test {request.node.parent.name}::{request.node.name}")
+        if request.config.getoption("--test-py-init"):
+            pytest.fail(reason="Scylla appears to have crashed")
+        else:
+            pytest.exit(f"Scylla appears to have crashed in test {request.node.parent.name}::{request.node.name}")
```

Comment:
I think these lines with failing test correctly can be added in dynamodb() as well, until I'm missing something. I know there were on in original code, but seem it's reasonable to add them.

Reply:
Removed this logic from the PR